Caramelo18/Visual-Kur
# Text editor

## What was added
1. Changed layout to a grid system (sidebar width still needs to be fixed)
2. Implemented a text editor using React-Ace (Ace text editor for React)
3. Added a toggle to turn on and off the text editor

### Added dependencies
- react-ace (v1.0, examples and documentation [here](https://github.com/securingsincity/react-ace))

## How to Test
1. Open the app
2. Click on Upload button
3. See the loaded file on text editor
4. Change the code on the text editor
5. See the changes reflected on the visual representation

## Notes for the reviewer
The text editor provides the undo functionality which can be tested using Ctrl+Z
The way the files were loaded was slightly modified to support this integration

## Screenshot
![captura de ecra de 2018-05-28 12-02-56](https://user-images.githubusercontent.com/12110089/40611653-228ea1c4-626f-11e8-97e7-0fb997ef6236.png)
